yt/yt/server/master/chunk_server/config.cpp

```diff
@@ -567,6 +567,9 @@ void TDynamicChunkManagerConfig::Register(TRegistrar registrar)
     registrar.Parameter("deprecated_codec_name_to_alias", &TThis::DeprecatedCodecNameToAlias)
         .Default();
 
+    registrar.Parameter("forbidden_erasure_codecs", &TThis::ForbiddenErasureCodecIds)
```

Comment:
1. Afaik deprecated is confusing naming for this option. For some erasure codecs we don't have convenient hardware. We might have it in the future. So it's not deprecated. Forbidden looks better here.
2.  Consistency was the reason I had Ids suffix for erasure initially. So which one of this naming conventions it is?
3. I am a bit worried about changing system options that we already have. I check that we don't have it on clusters I have access to, but in general I am not so sure. Aliases looks great, but this changes would make this PR less concise due to changes that are not related to original issue. May I change the naming of old options in a separate PR?

Reply:
> Afaik deprecated is confusing naming for this option. For some erasure codecs we don't have convenient hardware. We might have it in the future. So it's not deprecated. Forbidden looks better here.

Then let's rename the old flags from "deprecated" to "forbidden". This can be done via aliases, the way I have described above.

> Consistency was the reason I had Ids suffix for erasure initially. So which one of this naming conventions should I use?

Any one of them :-) Let's go with *_codecs, I suppose.

> May I change the naming of old options in a separate PR?
Sure.